directory/cli/src/utils.py

```diff
@@ -106,3 +107,6 @@ def display_success():
     command_string = " ".join(original_command().split(" ")[:2])
 #    command_string = " ".join([word.capitalize() for word in command_string])
     click.echo("------------- " + command_string + " successful -------------")
+
+def advanced_mode_enabled():
+    return (True if getenv('ADVANCED') == 'True' else False)
```

Comment:
This line could be simplified to not need the `if`/`else` I believe.

Could also be worth enabling advanced mode regardless of the case of `ADVANCED` - won't matter for entering this via the sandbox, but would allow users with direct access to the directory machine to do things like this to use the CLI in advanced mode outside the sandbox:
```bash
$ export ADVANCED=true
$ # or even: export ADVANCED=tRuE
$ directory user create tonkers --first t --last onkers
$ # etc.
```
 
Combining these suggestions:
```suggestion
    return getenv('ADVANCED').lower() == 'true'
```
:slightly_smiling_face: